source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -166,219 +255,227 @@ function New-StigCheckList
-                }
-            }
-            elseif ($PSCmdlet.ParameterSetName -eq 'result')
-            {
-                $originalSetting = Get-SettingsFromResult -DscResult $dscResult -id $convertedRule.DuplicateOf
+                    $originalSetting = Get-SettingsFromMof -ReferenceConfiguration $ReferenceConfiguration -Id $convertedRule.DuplicateOf
 
-                if ($originalSetting.InDesiredState -eq 'True')
-                {
-                    $status = $statusMap['NotAFinding']
-                    $findingDetails = 'See ' + $convertedRule.DuplicateOf + ' for Finding Details.'
-                    $comments = 'Managed via PowerStigDsc - this rule is a duplicate of ' + $convertedRule.DuplicateOf
+                    if ($originalSetting)
+                    {
+                        $status = $statusMap['Open']
+                        $findingDetails = 'See {0} for Finding Details.' -f $convertedRule.DuplicateOf
+                        $comments = 'Managed via PowerStigDsc - this rule is a duplicate of {0}' -f $convertedRule.DuplicateOf
+                    }
                 }
-                else
+                elseif ($PSCmdlet.ParameterSetName -eq 'dsc')
                 {
-                    $status = $statusMap['Open']
-                    $findingDetails = 'See ' + $convertedRule.DuplicateOf + ' for Finding Details.'
-                    $comments = 'Managed via PowerStigDsc - this rule is a duplicate of ' + $convertedRule.DuplicateOf
+                    $originalSetting = Get-SettingsFromResult -DscResults $DscResults -id $convertedRule.DuplicateOf
+
+                    if ($originalSetting.InDesiredState -eq 'True')
+                    {
+                        $status = $statusMap['NotAFinding']
+                        $findingDetails = 'See {0} for Finding Details.' -f $convertedRule.DuplicateOf
+                        $comments = 'Managed via PowerStigDsc - this rule is a duplicate of {0}' -f $convertedRule.DuplicateOf
+                    }
+                    else
+                    {
+                        $status = $statusMap['Open']
+                        $findingDetails = 'See {0} for Finding Details.' -f $convertedRule.DuplicateOf
+                        $comments = 'Managed via PowerStigDsc - this rule is a duplicate of {0}' -f $convertedRule.DuplicateOf
+                    }
                 }
             }
-        }
 
-        $writer.WriteStartElement("STATUS")
-        $writer.WriteString($status)
-        $writer.WriteEndElement(<#STATUS#>)
+            $writer.WriteStartElement("STATUS")
+            $writer.WriteString($status)
+            $writer.WriteEndElement(<#STATUS#>)
 
-        $writer.WriteStartElement("FINDING_DETAILS")
-        $writer.WriteString($findingDetails)
-        $writer.WriteEndElement(<#FINDING_DETAILS#>)
+            $writer.WriteStartElement("FINDING_DETAILS")
+            $findingDetails = ConvertTo-SafeXml -unescapedXmlString $findingDetails
```

Comment:
unescapedXmlString should start with an upper case "U".  I'll make a note of it on the function definition in this code review as well.

Reply:
Done.